src/View/Home/index.html.twig

```diff
@@ -1,5 +1,30 @@
 {% extends 'layout.html.twig' %}
 
+{% block title %} Home {% endblock %}
+
+{% block stylesheet %}
+<link rel="stylesheet" href="/assets/css/home.css"> 
+<link rel="stylesheet" href="/assets/css/home_tablet_and_desktop.css"> 
+
+{% endblock %}
+
 {% block content %}
-    <h1>Hello Wilder !</h1>
+        <nav class="nav-home">
+            {#to link a page, add url like example /file/page#}
+            <a href="#" class="questionmark-button"><img src="/assets/images/mobile/question_mark.png" alt="question mark" class="questionmark-icon"></a>
+            <a href="#" class="music-button"><img src="/assets/images/mobile/music_icon.png" alt="music icon" class="music-icon"></a>    
+        </nav>
+    <main>
+        <div class="castle">
+            <picture>
+                <source srcset="/assets/images/desktop/home_castle.png" media="(min-width: 530px)">
```

Comment:
Looks to me that the source tag can be deleted. 
In my opinion the game will always choose to show the image inside the img tag.
Furthermore the media attribute inside the source tag needs a "href" attribute in order to work which is in our case non existant (cf https://www.w3schools.com/tags/att_a_media.asp).

Reply:
Hi Soufiane, actually the tag Picture need both source (which is choose by the web browser only if the media query is respected) and img src (which is choose in all other cases). This syntax correspond to the official documentation https://developer.mozilla.org/fr/docs/Web/HTML/Element/picture

Reply:
Thanks for the link. It's a nice idea after all! All good